### ISSUE TITLE
Unable to access auto-hidden taskbar when Playnite is maximized

### ISSUE THREAD

**Issue by JosefNemec** [JosefNemec/Playnite#1106]
https://www.reddit.com/r/playnite/comments/banv5k/unable_to_access_autohidden_taskbar_when_playnite/

**Comment by namandixit** [comment]
To add a bit more detail to this issue, when the Task Bar is set to auto-hide, it actually reduces its width to 1 pixel. When that 1 pixel line is hovered upon, the Task Bar shows itself. For some reason, the Playnite application covers that 1 pixel line and prevents the Taskbar from getting activated. Since the Playnite app uses a custom window frame, the way that is set up might be the culprit.

**Comment by ArtexJay** [comment]
Hope to see this being fixed soon

**Comment by JosefNemec** [comment]
I'll try to fix it in one of 7.x updates (sadly it won't make 7.0 release). It's surprisingly not an easy fix.

**Comment by JosefNemec** [comment]
Status update: I spent roughly two days on this with no luck. I think somebody else will need to look at this because I've run out of ideas. When I finally got it working then top of the window would sometimes look like this:

![image](https://user-images.githubusercontent.com/3874087/75996388-15e11c80-5efe-11ea-9e68-69893e26bbf1.png)

I tried several fixes that [ControlzEx](https://github.com/ControlzEx/ControlzEx) and [MahApps.Metro](https://github.com/MahApps/MahApps.Metro) projects used in past (they had the same issue), with no luck. I don't know what we do different that it's not working properly in Playnite.
